docs/platforms/ruby/common/configuration/options.mdx

```diff
@@ -77,193 +141,191 @@ config.breadcrumbs_logger = [:active_support_logger]
 
-```ruby
-# send 50% of events
-config.sample_rate = 0.5
-```
+</ConfigKey>
 
-`send_client_reports`
+<ConfigKey name="trace-propagation-targets">
 
-Attach diagnostic client reports about dropped events to an existing envelope max once every 30s. Default is `true`.
+An optional property that controls which downstream services receive tracing data, in the form of a `sentry-trace` and a `baggage` header attached to any outgoing HTTP requests.
 
-This information will not be visible to users at the moment, but we're planning to add this information to user-facing UI.
+The option may contain an array of strings or regex against which the URLs of outgoing requests are matched.
+If one of the entries in the list matches the URL of an outgoing request, trace headers will be attached to that request.
+String entries do not have to be full matches, meaning the URL of a request is matched when it _contains_ a string provided through the option.
 
-If you **do not** want to send this data, you can opt-out by setting
+By default, trace headers are attached to every outgoing request from the instrumented client.
+
+</ConfigKey>
+
+<ConfigKey name="propagate-traces">
+
+By default, Sentry injects `sentry-trace` and `baggage` headers to outgoing requests made with `Net::HTTP` to connect traces between services. You can disable this behavior with
 
 ```ruby
-config.send_client_reports = false
+config.propagate_traces = false
 ```
 
-`send_default_pii`
+</ConfigKey>
 
-When its value is `false` (default), sensitive information like
+## Hooks
 
-- user ip
-- user cookie
-- request body
-- query string in the url
+These options can be used to hook the SDK in various ways to customize the reporting of events.
 
-will not be sent to Sentry.
+<ConfigKey name="before-send">
 
-You can re-enable it by setting:
+Provide a lambda or proc that is called with an SDK-specific message or error event object, and can return a modified event object, or `nil` to skip reporting the event. This can be used, for instance, for manual PII stripping before sending.
 
-```ruby
-config.send_default_pii = true
-```
+By the time <PlatformIdentifier name="before-send" /> is executed, all scope data has already been applied to the event. Further modification of the scope won't have any effect.
 
-`send_modules`
+<PlatformContent includePath="configuration/before-send/" />
 
-A boolean to decide whether to send modules (dependencies) information to Sentry (default is `true`).
+</ConfigKey>
 
-```ruby
-config.send_modules = false # if you don't want to send all the dependency info
-```
+<ConfigKey name="before-send-transaction">
 
-`skip_rake_integration`
+Provide a lambda or proc that is called with an SDK-specific transaction event object, and can return a modified transaction event object, or `nil` to skip reporting the event. One way this might be used is for manual PII stripping before sending.
```

Comment:
```suggestion
Provides a lambda or proc that's called with an SDK-specific transaction event object, and can return a modified transaction event object, or `nil` to skip reporting the event. One way this might be used is for manual PII stripping before sending.
```

Reply:
this isn't correct because the option doesn't do the providing, the user does